RecoLocalCalo/HcalRecProducers/src/HcalHitSelection.cc

```diff
@@ -191,6 +190,9 @@ void HcalHitSelection::produce(edm::Event& iEvent, const edm::EventSetup& iSetup
   skim(hf, *hf_out);
   iEvent.put(std::move(hf_out), hfTag.label());
 
+  if (!iEvent.getByToken(tok_ho_, ho))
+    return;
+  iEvent.getByToken(tok_ho_, ho);
```

Comment:
I wonder about these lines, as I have not seen such a pattern before. Is it necessary to call `getByToken` twice with the same arguments? Can the result of the first call not be checked, and passed on?